src/static/js/demo.js

```diff
@@ -13,17 +13,55 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { documentReady, handleFileSelect, startBpmnVisualization } from '../../index.es.js';
+import { documentReady, handleFileSelect, startBpmnVisualization, FitType, updateFitType } from '../../index.es.js';
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 function startDemo() {
-  startBpmnVisualization({ container: 'graph' });
+  startBpmnVisualization({ container: 'graph', loadOptions: { fitType: FitType[fitTypeSelected.value] } });
   document.getElementById('bpmn-file').addEventListener('change', handleFileSelect, false);
+  fitTypeSelected.addEventListener('change', updateFitTypeSelection, false);
 
   const parameters = new URLSearchParams(window.location.search);
   if (!(parameters.get('hideControls') === 'true')) {
     document.getElementById('controls').classList.remove('hidden');
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function updateFitTypeSelection(event) {
+  updateFitType(event);
+
+  if (event.target.value === 'None') {
+    resetClass(container);
+  } else {
+    setFixedSizeClass(container);
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function setFixedSizeClass(htmlElementId) {
+  const htmlElement = document.getElementById(htmlElementId);
+  htmlElement.classList.add('fixed-size');
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function resetClass(htmlElementId) {
+  const htmlElement = document.getElementById(htmlElementId);
+  htmlElement.classList.remove('fixed-size');
+}
+
+// Update the selected option at the initialization
+const parameters = new URLSearchParams(window.location.search);
+const parameterFitType = parameters.get('fitType');
+
+const fitTypeSelected = document.getElementById('fitType-selected');
+if (parameterFitType) {
+  fitTypeSelected.value = parameterFitType;
+}
+
+if (fitTypeSelected.value !== 'None') {
+  setFixedSizeClass('graph');
+}
```

Comment:
:warning: this code should be in the startDemo function for consistency with the rest of the configuration and to update the elements only when the lib is loaded.
I see some initialization errors when just moving the code to the function, and I don't have a quick solution for this now

Reply:
Done